tests/api/core/content-manager/index.test.api.js

```diff
@@ -273,6 +273,18 @@ describe.skip('Relations', () => {
       expect(tags.length).toBe(3);
     });
 
+    test('Get tags count in articles list', async () => {
+      const { body } = await rq({
+        url: '/content-manager/collection-types/api::article.article',
+        method: 'GET',
+      });
+
+      expect(body.results).toMatchObject([
+        { title: 'Article 1', tags: { count: 3 } },
+        { title: 'Article 2', tags: { count: 0 } },
+      ]);
```

Comment:
Ideally I'd want to test that the count doesn't change when publishing an unpublishing Tags. But Strapi doesn't seem to pick up when I enable D&P in the Tag schema in `packages/utils/api-tests/generators.js`